app/calculadora.py

```diff
@@ -87,6 +92,66 @@ def _create_input(self, master):
         self._entrada.insert(0,0)
         self._entrada.pack()
 
+
+    def _create_menu(self, master):
+        self.master.option_add('*tearOff', FALSE)
+        calc_menu = Menu(self.master)
+        self.master.config(menu=calc_menu)
+
+        #Configuração
+        config = Menu(calc_menu)
+        theme = Menu(config)
+        #Menu tema
+        config.add_cascade(label='Tema', menu=theme)
+        theme.add_command(label='Dark', command=self._set_theme_Dark)
+        theme.add_command(label='White', command=self._set_theme_White)
+        theme.add_command(label='Dracula', command=self._set_theme_Dracula)
+        """
+        Mac Os theme not configured
+        theme.add_command(label='MacOS', command=self._set_theme_MacOS)
+        """
+        theme.add_command(label='Dark Blue', command=self._set_theme_Dark_Blue)
+        #Configuração
+        config.add_separator()
+        config.add_command(label='Salir', command=self._exit)
+        calc_menu.add_cascade(label='Configuração', menu=config)
+        
+    def _set_theme_Dark(self):
+        self.settings['current_theme'] = "Dark"
+        print(self.settings)
+        with open('./app/settings/settings.json', 'w') as outfile:
+            json.dump(self.settings, outfile)
```

Comment:
E também não esqueça de escrever no arquivo colocando a indentação, isso facilita a leitura do arquivo posteriormente.

4 espaços já é suficiente:
```suggestion
            json.dump(self.settings, outfile, indent=4)
```